### ISSUE TITLE
Some tests fail, using opencv4tegra on tx1

### ISSUE THREAD

**Issue by bpinaya** [NVIDIA/caffe#294]
Hi there, quick question, while running the tests in this caffe fork out of 
[==========] 2122 tests, 6 seemed to fail:
`[  FAILED  ] 6 tests, listed below:
[  FAILED  ] DetectNetTransformationLayerTest/3.TestNoAugmentation, where TypeParam = caffe::GPUDevice<double>

[  FAILED  ] DetectNetTransformationLayerTest/3.TestScaleDown, where TypeParam = caffe::GPUDevice<double>

[  FAILED  ] DetectNetTransformationLayerTest/3.TestFlip, where TypeParam = caffe::GPUDevice<double>

[  FAILED  ] DetectNetTransformationLayerTest/3.TestRotation, where TypeParam = caffe::GPUDevice<double>

[  FAILED  ] DetectNetTransformationLayerTest/3.TestScaleUp, where TypeParam = caffe::GPUDevice<double>

[  FAILED  ] DetectNetTransformationLayerTest/3.TestDesaturation, where TypeParam = caffe::GPUDevice<double>`

From what I could look somewhere else, I might have to deal with opencv, in this case opencv4tegra.
I am yet to test if this will affect the DetectNet examples from Dustin, I'll update this as required.

**Comment by m-khurram-amin** [comment]
Similar issue ; One failed test is common as above while other 2 are related to same **DetectNetTransformationLayerTest**
I am using OpenCV 3.2 and I have compiled it myself. BVLC Caffe builds perfectly, without any test case failure.
--------------------------------------------------------------------------------------------------------------------
```
[----------] Global test environment tear-down
[==========] 2122 tests from 287 test cases ran. (315237 ms total)
[  PASSED  ] 2119 tests.
[  FAILED  ] 3 tests, listed below:
[  FAILED  ] DetectNetTransformationLayerTest/1.TestAllAugmentation, where TypeParam = caffe::CPUDevice<double>
[  FAILED  ] DetectNetTransformationLayerTest/1.TestHueRotation, where TypeParam = caffe::CPUDevice<double>
[  FAILED  ] DetectNetTransformationLayerTest/1.TestDesaturation, where TypeParam = caffe::CPUDevice<double>

 3 FAILED TESTS
Makefile:535: recipe for target 'runtest' failed
make: *** [runtest] Error 1
```
-------------------------------------------------------------------------------------------------------------------------

**Comment by drnikolaev** [comment]
fixed in 0.16
